integration-tests/realm-apps/pv-int-tests/services/mongodb/rules/test_data.Dog.json

```diff
@@ -1,35 +0,0 @@
-{
-    "collection": "Dog",
-    "database": "test_data",
-    "roles": [
-        {
-            "name": "default",
-            "apply_when": {},
-            "write": true,
-            "insert": true,
-            "delete": true,
-            "additional_fields": {}
-        }
-    ],
-    "schema": {
-        "properties": {
-            "_id": {
-                "bsonType": "objectId"
-            },
-            "breed": {
-                "bsonType": "string"
-            },
-            "name": {
-                "bsonType": "string"
-            },
-            "realm_id": {
-                "bsonType": "long"
-            }
-        },
-        "required": [
-            "_id",
-            "name"
-        ],
-        "title": "Dog"
-    }
```

Comment:
These schemas are probably not needed anymore, since the apps are in dev-mode. But I wanted to ensure that it wasn't just deleted by accident? 🙂

Reply:
I decided to be bold and see if the test apps functioned and passed without these schemas.  They did 💪🏼  So they have been removed to prevent confusion.